streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStateUpdateTask.java

```diff
@@ -69,7 +69,7 @@ public GlobalStateUpdateTask(final LogContext logContext,
         final Map<String, String> storeNameToTopic = topology.storeToChangelogTopic();
         for (final String storeName : storeNames) {
             final String sourceTopic = storeNameToTopic.get(storeName);
-            final SourceNode<?, ?, ?, ?> source = topology.source(sourceTopic);
+            final SourceNode<?, ?> source = topology.source(sourceTopic);
```

Comment:
I forget why now, but it was necessary to revise the generic params on source and sink nodes. Source nodes are always just pass-thoughs, and sink nodes can't forward, so their output types are Void. In both cases, it means they only need two generic parameters now, so I had to change like 100 declarations like this.